### DISCUSSION TITLE
The links which assets are served are not working when run packer start

### DISCUSSION THREAD

**Discussion by CodingRedStar** [hayes0724/shopify-packer#74] [Q&A]
I did run packer start and everything is done successfully without any errors.
But local server styling is still broken. As I checked, the server which the assets are served is not working. When I link to that url, it is still loading and not available to connect there.
But if I run `packer start --skipFirstDeploy`, it is working but anyway still assets server is not available to connect.
Hope to hear about that solution asap. 
Thanks.

**Comment by mrpacman101** [comment]
@CodingRedStar 
I think you can close this issue since it was not related with shopify packer.
